WpfMath/XmlUtilities.cs

```diff
@@ -41,7 +42,7 @@ public static double AttributeDoubleValue(this XElement element, string attribut
                     return defaultValue.Value;
                 throw new InvalidOperationException();
             }
-            return double.Parse(attribute.Value);
+            return double.Parse(attribute.Value, CultureInfo.InvariantCulture);

```

Comment:
Could you please do the same for `int.Parse` in the previous method? I believe it could lead to the same problems in some phantasmagorical circumstances.